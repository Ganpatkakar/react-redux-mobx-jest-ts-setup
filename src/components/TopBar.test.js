import { TopBar } from "./TopBar";
import { shallow } from "enzyme";

test("topbar component", () => {
  const component = shallow(<TopBar />);
  expect(component).toMatchSnapshot();
});
