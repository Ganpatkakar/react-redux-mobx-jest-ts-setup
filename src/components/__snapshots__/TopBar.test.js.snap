// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`topbar component 1`] = `
<ul>
  <li>
    <Link
      to="/"
    >
      Home
    </Link>
  </li>
  <li>
    <Link
      to="/about"
    >
      About
    </Link>
  </li>
  <li>
    <Link
      to="/dashboard"
    >
      Dashboard
    </Link>
  </li>
</ul>
`;
